login-workflow/src/components/StepperCard/StepperCard.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactNode } from 'react';
+import Card, { CardProps } from '@mui/material/Card';
+import { SxProps, Theme, useTheme } from '@mui/material/styles';
+import { Spinner } from '../Spinner';
+import defaultBackgroundImage from '../../assets/images/background.svg';
+import { useInjectedUIContext } from '@brightlayer-ui/react-auth-shared';
+import Box, { BoxProps } from '@mui/material/Box';
+
+export type StepperCardProps = BoxProps & {
+    loading?: boolean;
+    backgroundImage?: string;
+    sx?: SxProps<Theme>;
```

Comment:
you may not need to define the `sx` prop here since you are getting it for free from the inherited `Box` component